numba/tests/gdb/test_pretty_print.py

```diff
@@ -0,0 +1,68 @@
+# NOTE: This test is sensitive to line numbers as it checks breakpoints
```

Comment:
The filename should be something like `test_gdb_pretty_print.py`

Reply:
I'm not sure about this, it's in the `gdb` module already and none of the other test files adjacent to this have `gdb` in their name. Further, these test files cannot be run directly, the `test_gdb_dwarf.py` test invokes them via a harness.

Reply:
ok, i thk i missed the `gdb` directory